src/storage/storage.ts

```diff
@@ -134,6 +161,26 @@ export class Storage implements IStorage {
     return this.store.getAddressAtIndex(index);
   }
 
+  /**
+   * Get the address public key, if not available derive from xpub
+   * @param {number} index
+   * @async
+   * @returns {Promise<string>} The public key DER encoded in hex
+   */
+  async getAddressPubkey(index: number): Promise<string> {
```

Comment:
Are you using this method somewhere?

Reply:
Not in the wallet-lib but HSM custom signature method will require this.